common/typographical-conventions.html

```diff
@@ -33,7 +33,8 @@
     and links to the references section.</dd>
   <dt class="changed">Changes from Recommendation</dt><dd>
     Sections or phrases changed from the previous Recommendation
-    are <span class="changed">highlighted</span>.</dd>
+    may be <span class="changed">highlighted</span> using a control
+    in <a href="#how-to-read-this-document" class="sectionRef"></a>.</dd>
 </dl>
 
```

Comment:
While we are at it... I just realized that the highlight backgrounds uses the same (or almost the same) color as the note backgrounds. This may be misleading. We may want to choose another color...

Reply:
Uses "lime" now. This is the same as the diff-changes use.